atrium-api/src/agent/store.rs

```diff
@@ -4,7 +4,8 @@ pub use self::memory::MemorySessionStore;
 pub(crate) use super::Session;
 use async_trait::async_trait;
 
-#[async_trait]
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
```

Comment:
It might be useful to mention this change (the `Send` bound on whatever this is effecting now being optional for `wasm32-*` targets) in the `atrium-api` changelog (for the public traits that are altered).